app_client/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
config API:
{
    "roads": "roads/",
    "cars": "cars/{id}/{model}/",
    "gas_station": "gas_stations/"
}


100

cars/${in}/{model}/create/
cars/${id}/{model}/update/
cars/${id}/{model}/move/{town_id}
cars/{id}/...

{
    "cars": 
    {
           "${id}":
           { 
                   "{model}":
                    {
                           "create": "create_car"
                            "move/{town_id}": "move_car_to_town"
                    }
           }
     }
}



config
config application:
test_config.json
dev_
prod_
{
      "genome_api_url": "https://spb-stu-rshiny-lab.herokuapp.com/api/",
      ""
      base_url
}